### PR TITLE
Remove unnecessary code block

### PR DESCRIPTION
**Pull request by rene-dekker**

### CODE REVIEW

File: pkg/controller/monitor/monitor_controller.go [hunk 1]
```diff
@@ -270,15 +270,6 @@ func (r *ReconcileMonitor) Reconcile(ctx context.Context, request reconcile.Requ
 		return reconcile.Result{}, err
 	}
 
-	if err != nil {
```
Comment (rene-dekker) [inline pkg/controller/monitor/monitor_controller.go:273]: err is always nil at this point.

**Review by caseydavenport — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**